tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = lint, pylint
 skip_missing_interpreters = True
 
-[testenv]
+[testenv:pytest]
```

Comment:
Can we keep this named [testenv]? That will line up with some other changes I was planning to make to our testing/linting setup.

Reply:
Sure. That was the only way that I could figure out how to get pytest to run in the Travis CI build with the existing Tox configuration. Could you suggest how to get that working with the original configuration here?

Reply:
Try adding py35 to the envlist.

Reply:
Let's try making it look like this:

```ini
[testenv]
basepython = {env:PYTHON3_PATH:python3}
commands =
     pytest {posargs}
     {toxinidir}/script/check_dirty
deps =
     -r{toxinidir}/requirements_test.txt
     pytest-xdist
```

Reply:
Scratch that, you found the issue.